src/com/fedorowiat/menu/LoginMenu.java

```diff
@@ -15,13 +17,22 @@ public LoginMenu() {
         this.input = new Scanner(System.in);
     }
 
-    public void showLoginMenu() {
+    public void showLoginMenu(DataBase dataBase) {
         System.out.print("\n\nPodaj login: ");
-        String login = input.nextLine();
-
+        var login = input.nextLine();
         System.out.print("Podaj hasło: ");
-        String password = input.nextLine();
+        var password = input.nextLine();
         System.out.println("\n\n");
+            if(dataBase.checkLoginData(login, password)){
+                if (dataBase.getAccessLevel(login, password).equals("false")){
```

Comment:
mozesz sobie do zmiennej zapisac wynik getAccessLevel, bo tak to 2x baze odpytujesz o ta sama zmienna.